types/packageMeta.ts

```diff
@@ -1,11 +1,19 @@
+export interface Author {
+  name: string;
+  avatar?: string;
+  email?: string;
+}
+
 export interface PackageMetaInterface {
   latest: {
+    author?: Author;
```

Comment:
Author can be either a string or an object

Reply:
Yes, the metadata is quite flexible, https://docs.npmjs.com/files/package.json#people-fields-author-contributors you can see the different shape of `Author`. 

Here an example. https://github.com/verdaccio/monorepo/blob/master/core/types/index.d.ts#L39

You might use `import {Author} from `@verdaccio/types` as well, so  you don't have to duplicate the types.

Reply:
I had a go using the type from the types package - unfortunately it's missing the `avatar` property so it causes more type errors.

I also tried rewriting as `string | Author` - it also causes a number of type errors (e.g. the `Author` component assumes `author` is an object and doesn't handle parsing the string version).

Since it's an existing typing bug, would you be OK considering it outside the scope of this PR?

Reply:
Yes, make it green and we will fix it later. I will write it down. Thanks for the research.